Makefile.docker

```diff
@@ -73,3 +78,11 @@ eksctl-image: intermediate-image ## Build the eksctl image that has release arte
 	      --iidfile=.eksctl_image.iid \
 	      --tag="$(eksctl_image_name)" -
 
+.PHONY: test build
+build test: ## Build and test using the build image
```

Comment:
test build or build test?

Reply:
If this is correct go ahead and merge

Reply:
The order doesn't matter here, but I'll make it same the line above for cosmetic reasons.